### DISCUSSION TITLE
Création d'icone

### DISCUSSION THREAD

**Discussion by Tilania** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#236] [Q&A] [answered]
Bonjour,

Comment dois-je faire pour créer une nouvelle icone à proposer lors de la pull request de ma fonctionnalité ? Via une IA ou je dois demander à quelqu'un pour la faire ?

Merci :)

**Comment by Mcfloy** [comment, marked as answer]
Idéalement sur https://icones8.fr/ tu cherches un icône dans le style `iOS 16 Filled` ou `Material Filled`

Si l'icône n'y est pas tu peux utiliser le discord soz midjourney: https://discord.gg/SHWphje56A
